docs/ml-features.md

```diff
@@ -459,6 +459,42 @@ column, we should get the following:
 "a" gets index `0` because it is the most frequent, followed by "c" with index `1` and "b" with
 index `2`.
 
+Additionaly, there are two strategies regarding how `StringIndexer` will handle
+unseen labels when you have set up a `StringIndexer` on a dataset which you want
```

Comment:
"set up" --> "fit"
"on a dataset which you want to reuse on another" --> "on one dataset and then use it to transform another dataset"